File: AtCoder/abc211/A.swift
private func readLn() -> String {
	return readLine() ?? ""
}

private func readLns() -> [String] {
	return readLn().split(separator: " ").map{ String($0) }
}

private func readDoubles() -> [Double] {
	return readLns().compactMap{ Double($0) }
}

let ab = readDoubles()
let ans = ((ab[0] - ab[1]) / 3.0) + ab[1]
print(ans)

File: AtCoder/abc211/A.kt
private fun readLn() = readLine() ?: ""
private fun readLns() = readLn().split(" ")
private fun readDoubles() = readLns().map { it.toDouble() }

fun main() {
    val ab = readDoubles()
    print(((ab[0] - ab[1]) / 3.0) + ab[1])
}